source/plugin/Assets/GoogleMobileAds/Platforms/Unity/BannerClient.cs

```diff
@@ -154,6 +154,7 @@ public void DestroyBannerView()
         {
```

Comment:
Can you clean up the indentation here? It looks like the {'s are way out of line with where they belong. I see this isn't the diff, but since you're here ...